doc/user_guide/marks/geoshape.rst

```diff
@@ -0,0 +1,689 @@
+In the following example the input geometry is not projected and is instead rendered directly in raw coordinates using the ``identity`` projection type. We have to define the ``reflectY`` as well since Canvas and SVG treats positive ``y`` as pointing down.
+
+.. altair-plot::
+
+    alt.Chart(gdf_sel).mark_geoshape().project(
+        type='identity', 
+        reflectY=True
+    )
+
+Mapping Polygons
+~~~~~~~~~~~~~~~~
+The following example maps the visual property of the ``name`` column using the ``color`` encoding.
+
+.. altair-plot::
+
+    alt.Chart(gdf_sel).mark_geoshape().encode(
+        color='name:N'
+    )
+
+Since each country is represented by a (multi)polygon, we can separate the ``stroke`` and ``fill`` definitions as such: 
+
+.. altair-plot::
+
+    alt.Chart(gdf_sel).mark_geoshape(
+        stroke='white',
+        strokeWidth=1.5
+    ).encode(
+        fill='name:N'
+    ) 
+
+Mapping Lines
+~~~~~~~~~~~~~
+By default Altair assumes for ``mark_geoshape`` that the mark's color is used for the fill color instead of the stroke color.
+This means that if your source data contain (multi)lines, you will have to explicitly define the ``filled`` value as ``False``.
+
+Compare:
+
+.. altair-plot::
+    
+    gs_line = gpd.GeoSeries.from_wkt(['LINESTRING (0 0, 1 1, 0 2, 2 2, -1 1, 1 0)'])
+    alt.Chart(gs_line).mark_geoshape().project(
+        type='identity', 
+        reflectY=True
+    )    
+
+With:
+
+.. altair-plot::
+    
+    gs_line = gpd.GeoSeries.from_wkt(['LINESTRING (0 0, 1 1, 0 2, 2 2, -1 1, 1 0)'])
+    alt.Chart(gs_line).mark_geoshape(
+        filled=False
+    ).project(
+        type='identity', 
+        reflectY=True
+    ) 
+
+Using this approach one can also style Polygons as if they are Linestrings:
+
+.. altair-plot::
+
+    alt.Chart(gdf_sel).mark_geoshape(
+        filled=False,
+        strokeWidth=1.5
+    ).encode(
+        stroke='name:N'
+    )
+
+Mapping Points
+~~~~~~~~~~~~~~
+Points can be drawn when they are defined as ``Points`` within a GeoDataFrame using ``mark_geoshape``.
+We first assign the centroids of Polygons as Point geometry and plot these:
+
+.. altair-plot::
+
+    gdf_centroid = gpd.GeoDataFrame(
+        data=gdf_sel.drop('geometry', axis=1), 
+        geometry=gdf_sel.geometry.centroid
+    )
+
+    alt.Chart(gdf_centroid).mark_geoshape()
+
+
+Caveat: To use the ``size`` encoding for the Points you will need to use the ``mark_circle`` in combination with the ``latitude`` and ``longitude`` encoding channel definitions.
+
+.. altair-plot::
+
+    gdf_centroid['lon'] = gdf_centroid['geometry'].x
+    gdf_centroid['lat'] = gdf_centroid['geometry'].y
+
+    alt.Chart(gdf_centroid).mark_circle().encode(
+        longitude='lon:Q',
+        latitude='lat:Q',
+        size="pop_est:Q"
+    )
+
+You could skip the extra assignment to the ``lon`` and ``lat`` column in the GeoDataFrame and use the coordinates directly.  We combine the chart with a basemap to bring some perspective to the points:
```

Comment:
There has been an issue on this, https://github.com/altair-viz/altair/issues/2203, therefor I thought to include the approach. Result is the same. Probably better to remove this section in order to avoid confusion, technically it is not related to mark_geoshape as well.

Reply:
I mean I do like that we show nested property access with the `. ` because I don't think that is explained anywhere else. Maybe we just need to change the text to more clearly motivates what is going on here? (or show an example of nested property elsewhere of you still think it's better to remove this)

Reply:
Yes, maybe change it into a note here or elsewhere, Will think about it.